### PR TITLE
finally normally added a README.md file

### PR DESCRIPTION
**Pull request by xpolb01**

### CODE REVIEW

**Comment by criticus** [comment]
:+1:

**Comment by criticus** [comment]
@superkunya please review

**Comment by criticus** [comment]
@xpolb01

**Comment by criticus** [comment]
:panda_face:

**Comment by xpolb01** [comment]
:+1:

**Comment by superkunya** [comment]
:+1:

**Comment by xpolb01** [comment]
:8ball:

File: README.md [hunk 1]
```diff
@@ -0,0 +1,4 @@
+README.md
+
+
```
Comment (criticus) [inline README.md:3]: why do you use so mane line breaks? consider to remove at least a few of them

**Pull request merged**